cocoon_data/__init__.py

```diff
@@ -1,18625 +0,0 @@
-import os
```

Comment:
are we going to move this to a different folder?

Reply:
Yes, I'll extract the core logic we need and modularize it.

Reply:
ok, then lets make part of the same PR